yabt/docker.py

```diff
@@ -564,6 +564,9 @@ def format_docker_cmd(docker_cmd):
         })
     # Generate ybt_bin scripts
     if ybt_bin_path:
+        if ybt_bin_path.startswith('//'):
```

Comment:
what's the use case of creating the ybt_bin under a completely different path from the related build target?

Reply:
@itamaro We are writing a DockerImage target and it seems that it doesn't generate a ybt_bin/.. script unless we use the ybt_bin_path argument (is that the expected behavior?) and currently the only path accepted as value of this arg is one which is a full path starting with the <workspace>/ybt_bin

Reply:
> is that the expected behavior?

yes. "plain" DockerBuilder doesn't automatically know how to generate a ybt_bin (if it's even relevant) - it's the PythonApp / CppApp wrappers that bring in that knowledge.

if you follow the stack of {Python,Cpp}App builders, you'll see [this](https://github.com/resonai/ybt/blob/master/yabt/builders/dockerapp.py#L57) builds the ybt_bin_path, so you *should* be able to use something similar directly - if not, then it's indeed a bug.

Reply:
As you said, I didn't add ybt_bin output as default but also added support for ybt_bin_path='.' as a way to have it in the default location